analyzers/validator/dummy/analyzer_toml1.toml

```diff
@@ -0,0 +1,14 @@
+name = "Todo comments checker"
```

Comment:
Rename `dummy` to `testdata`. Separate TOMLs into separate directories. For example, `testdata/analyzer1/.deepsource/analyzer.toml`.